tests/fail/concurrency/thread_local_static_dealloc.rs

```diff
@@ -1,13 +1,13 @@
-// ignore-windows: Concurrency on Windows is not supported yet.
-
 //! Ensure that thread-local statics get deallocated when the thread dies.
 
 #![feature(thread_local)]
 
 #[thread_local]
 static mut TLS: u8 = 0;
 
-fn main() { unsafe {
-    let dangling_ptr = std::thread::spawn(|| &TLS as *const u8 as usize).join().unwrap();
-    let _val = *(dangling_ptr as *const u8); //~ ERROR dereferenced after this allocation got freed
-} }
+fn main() {
+    unsafe {
+        let dangling_ptr = std::thread::spawn(|| &TLS as *const u8 as usize).join().unwrap();
+        let _val = *(dangling_ptr as *const u8); //~ ERROR dereferenced after this allocation got freed
+    }
+}
```

Comment:
There are a lot of spurious diffs here. Looks like you formatted those files?

They have been formatted on master in the mean time. So if you could rebase (which will be painful due to the double-format...), that'd be good as it would focus the diff on the relevant parts.

Reply:
The rebase is done.